src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -2,13 +2,16 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
-   href="${publishButton.actionLink}"
+<%@ attribute name="showButtonAndTooltip" required="true" %>
+<a href="${publishButton.actionLink}" 
+   class="<c:if test="${showButtonAndTooltip}">btn ${buttonType}</c:if> session-${publishButton.actionNameLowercase}-for-test"
```

Comment:
`btn` can be moved to the `buttonType` of sessionPanel.tag. Then, you can remove the if check if you have set `buttonType=""` in feedbackSessionActions.tag and 
`buttonType="btn ${sessionPanel.feedbackSessionPublishButton.buttonType}"` in sessionPanel.tag.